### ISSUE TITLE
ATS 9.1.3 build failures on Debian

### ISSUE THREAD

**Issue by jbfavre** [apache/trafficserver#9088]
Hello,
I've recently been aware of build failure on Debian, which I can not explain for now.

Bug report is available at: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1019553
Debian build log on arm64 is also available: https://bugs.debian.org/cgi-bin/bugreport.cgi?att=1;bug=1019553;filename=trafficserver_arm64.build;msg=5

Looks like similar to #8848: test  `ts_file_io` which is part of `test_tscore`.

**Comment by traeak** [comment]
Likely you are seeing this with 22.04?  Both 20.04 and 21.04 build fine.  There are some patches that are in master, 10-Dev and 9.2.x which haven't been back ported which make ATS compatible with newer versions of openssl.

**Comment by maskit** [comment]
I installed Debian testing and it worked for me. Although I saw many warnings while I'm building ATS, the test passed successfully.

The error code seems like ENOENT. I was able to see the error if I rename "/etc/hosts" to something else. Does the file exist on the build environment?

**Comment by jbfavre** [comment]
@traeak Thanks for your comment. I'm building ATS on Debian, not Ubuntu. Even if those two operating system are very similars, there're many differences that could make one succeed when the other one fails.
Anyway, I finally found a solution (see below)

@maskit Thanks for your help.
While investigating the failure, I came to the exact same conclusion: for any reason, `/etc/hosts` wasn't present in the build `chroot`.
As suggested in the Debian report, the solution is o update the build dependencies.

Again, thanks for your help, and sorry for the noise as the problem was on Debian side.

Best
